pkg/config/config.go

```diff
@@ -35,6 +36,10 @@ type Config struct {
 		GroupObjectType       string          `json:"group_object_type"`
 		IdentityObjectType    string          `json:"identity_object_type"`
 		IdentityRelation      string          `json:"identity_relation"`
+		Identity              struct {
+			ObjectType string
+			Relation   string
+		}
```

Comment:
This struct should have a ``json:"-"`` tag. We don't want to serialize or deserialize these fields.